.evergreen/config.yml

```diff
@@ -480,172 +480,168 @@ tasks:
 
 
 # Standard test tasks {{{
-    - name: "test-latest-replicaset"
+
+    - name: "test-2.4-standalone"
+      tags: ["2.4", "standalone"]
       commands:
         - func: "bootstrap mongo-orchestration"
           vars:
-            VERSION: "latest"
-            TOPOLOGY: "replica_set"
-            AUTH: ${AUTH}
-            SSL: ${SSL}
```

Comment:
The `AUTH: ${AUTH}` variable definitions are redundant. The [expansion docs](https://github.com/evergreen-ci/evergreen/wiki/Project-Files#expansions) sort of imply that once defined they can be used anywhere and my own changes on the Pymongo driver verify that.